i18n/en.i18n.json

```diff
@@ -274,6 +274,8 @@
   "Login_with" : "Login with %s",
   "login_with" : "Or login directly with",
   "Logout" : "Logout",
+  "Logout_Others" : "Logout Others",
```

Comment:
I wouldn't call it "Logout Others". Maybe make it more specific "Logout from other logged in locations", cause it might be confusing...